### PR TITLE
Use common Markdown for descriptions

### PR DESCRIPTION
**Pull request by RomanGotsiy**
There are a lot of different Markdown dialects. Let's stick to Common Markdown as it was recently done by GitHub and others.
I believe most of markdown renderers will support soon or already support Common Markdown.

### CODE REVIEW

File: spec/Section 4 -- Introspection.md [hunk 1]
```diff
@@ -74,9 +74,9 @@ underscores.
 
 All types in the introspection system provide a `description` field of type
 `String` to allow type designers to publish documentation in addition to
-capabilities. A GraphQL server may return the `description` field using Markdown
+capabilities. A GraphQL server may return the `description` field using [Common Markdown](http://spec.commonmark.org/)
```
Comment (wincent) [inline spec/Section 4 -- Introspection.md]: Seems like this long line would best be wrapped.

**Comment by wincent** [comment]
And for reference, I looked up [this GitHub blog post](https://github.com/blog/2333-a-formal-spec-for-github-flavored-markdown) on the subject of the Markdown spec.

File: spec/Section 4 -- Introspection.md (hunk 1 above)
Comment (RomanGotsiy) [inline spec/Section 4 -- Introspection.md]: done

**Comment by wincent** [comment]
Ok, going to apply a couple of tweaks and merge this (namely, that the spec is called "CommonMark" and doesn't refer to "Common Markdown"). Thanks!

**Pull request merged**